### ISSUE TITLE
Build failing: macro_reexport has been removed

### ISSUE THREAD

**Issue by denysvitali** [SergioBenitez/Rocket#626]
With the latest Rust nightly Rocket build fails.

```
   Compiling rocket_contrib v0.3.9
error[E0557]: feature has been removed
 --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/rocket_contrib-0.3.9/src/lib.rs:1:12
  |
1 | #![feature(macro_reexport)]
  |            ^^^^^^^^^^^^^^
  |
note: subsumed by `#![feature(use_extern_macros)]` and `pub use`
 --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/rocket_contrib-0.3.9/src/lib.rs:1:12
  |
1 | #![feature(macro_reexport)]
  |            ^^^^^^^^^^^^^^

error[E0658]: The attribute `macro_reexport` is currently unknown to the compiler and may have meaning added to it in the future (see issue #29642)
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/rocket_contrib-0.3.9/src/lib.rs:47:30
   |
47 | #[cfg_attr(feature = "json", macro_reexport(json_internal))]
   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(custom_attribute)] to the crate attributes to enable

error: aborting due to 2 previous errors

Some errors occurred: E0557, E0658.
For more information about an error, try `rustc --explain E0557`.
error: Could not compile `rocket_contrib`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

The problem is that `macro_reexport` feature has been removed (see https://github.com/rust-lang/rust/pull/49982)

**Comment by denysvitali** [comment]
There is already a PR available (https://github.com/SergioBenitez/Rocket/pull/624) and #625 is tracking this aswell, but was closed before the PR has been merged.

**Comment by SergioBenitez** [comment]
Fixed in v0.3.10. 🎉

**Comment by denysvitali** [comment]
Thank you! ❤️
